functions/HttpTriggerAPIUsers/__init__.py

```diff
@@ -3,12 +3,21 @@
 import logging
 import os
 import redis
+import urllib.parse as urlparse
+from urllib.parse import parse_qs
 import azure.functions as func
 
 USERS_CACHE_KEY = b'users:all'
 CACHE_TOGGLE = os.environ["CACHE_TOGGLE"]
 
 def main(req: func.HttpRequest) -> func.HttpResponse:
+    url = urlparse.urlparse(req.url)
+    count = parse_qs(url.query)['count']
+    page = parse_qs(url.query)['page']
```

Comment:
- where are you using count and page variables. 
- also, I think that would be a good idea to use an if statement to avoid crashing the app when count and page are not defined in url.